### ISSUE TITLE
Javadoc submission

### ISSUE THREAD

**Issue by kychanbx** [khwang0/2018F-COMP3111#85]
Can all team members submit the same java doc that consists of all functions we write? Since it is the individual task, we are not sure if we can submit the same javadoc.

**Comment by comp3111ta** [comment]
yes. But Marks calculates separately, depends on which function we are going to blame.
